app/controllers/api/v1/session.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class Session < Grape::API
+      desc 'Session related routes'
+      resource :sessions do
+        desc 'Start a new session'
+        params do
+          requires :email
```

Comment:
add `allow_blank: false` for all required params

Reply:
agree